person.rb

```diff
@@ -1,8 +1,9 @@
+# Person Class
 class Person
   attr_reader :id
   attr_accessor :name, :age
 
-  def initialize(id, name = "Unknown", age = 0)
+  def initialize(id, name = 'Unknown', age = 0)
```

Comment:
- [ ] I want to kindly suggest that for your constructor function, you do not need users to pass in an `id` rather this should automatically be generated when a new person is created, see the example code below for how to achieve this
- [ ] Also one more parameter you should requires to create a new person should be the `parent_permission` which by default should be equal to `true`
- [ ] Also the age parameter does  not require a default value
- >* example code 
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
 end
```